lib/workers/repository/dependency-dashboard.ts

```diff
@@ -6,6 +6,53 @@ import { getProblems, logger } from '../../logger';
 import { platform } from '../../platform';
 import { BranchConfig, BranchResult } from '../types';
 
+export interface DependencyDashboard {
+  dependencyDashboardChecks: Record<string, string>;
+  dependencyDashboardRebaseAllOpen: boolean;
+}
```

Comment:
Move to `./types.ts` if it will be used otherwise later?

or can we make it internal? It seems only be used in this file.
```suggestion
interface DependencyDashboard {
  dependencyDashboardChecks: Record<string, string>;
  dependencyDashboardRebaseAllOpen: boolean;
}
```